### PR TITLE
Updated counter for validation requests

### PR DESCRIPTION
**Pull request by benbaumann95**
### Description of change

- See commits

### Story Link

https://trello.com/c/hyFJgtEW/1021-show-a-newly-uploaded-replacement-document-as-updated-in-the-validation-tasklist

### CODE REVIEW

**Review by allbecauseyoutoldmeso — APPROVED** [review]
Looks good to me! Although have to admit that it's broken my brain a bit.

Just one issue - there's a weird 'Submodule bops-applications updated 25 files' thing in the middle which I think shouldn't be there 🤔

File: app/controllers/api/v1/replacement_document_validation_requests_controller.rb [hunk 1]
```diff
@@ -33,7 +33,8 @@ def show
       def update
         @replacement_document_validation_request = @planning_application.replacement_document_validation_requests.find_by(id: params[:id])
         new_document = @planning_application.documents.create!(file: params[:new_file])
-        @replacement_document_validation_request.update!(state: "closed", new_document: new_document)
+        @replacement_document_validation_request.update!(new_document: new_document)
+        @replacement_document_validation_request.close!
```
[the remark this answers is not in the document]
Comment (benbaumann95) [reply app/controllers/api/v1/replacement_document_validation_requests_controller.rb:37]: Yeah was going to leave this refactor to another PR
All of these updates need to be wrapped in a transaction and preferably moved to the model

**Comment by benbaumann95** [comment]
> Looks good to me! Although have to admit that it's broken my brain a bit.
> 
> Just one issue - there's a weird 'Submodule bops-applications updated 25 files' thing in the middle which I think shouldn't be there 🤔

Thanks Kate, it broke my brain too.
I think this logic will be nicer when we eventually do the other validation requests full refactor.
Or may just need another rethink because i'm not overly happy with it but don't really want to spend more time on it

**Review by EGiataganas — APPROVED** [review]
Good job @benbaumann95 !! I think it has 'Submodule bops-applications updated 25 files' as the last time we commit the latest update was 3 months ago! So it's fine 😄

**Pull request merged**